astroquery/esasky/core.py

```diff
@@ -529,12 +796,12 @@ def get_maps(self, query_table_list, missions=__ALL_STRING,
             log.info("No maps found.")
         return maps
 
-    def get_images(self, position, radius=__ZERO_ARCMIN_STRING,
+    def get_images(self, position="", radius=__ZERO_ARCMIN_STRING,
```

Comment:
add a note in the parameter list that `position` is now optional

Reply:
and maybe add a useful error message if a user fails to specify at least one of (position, observation_ids)